tests/wicked/before_test.pm

```diff
@@ -121,6 +120,7 @@ sub run {
         $package_list .= ' gcc'                         if check_var('WICKED', 'advanced');
         zypper_call('-q in ' . $package_list, timeout => 400);
         $self->reset_wicked();
+        record_info('PKG', script_output(q(rpm -qa 'wicked*' --qf '%{NAME}\n' | sort | uniq | xargs rpm -qi)));
```

Comment:
Do you intend to have `record_info` even when *check_var('IS_WICKED_REF', '1')*. if you want to have the info even then it seems that you should move the statement one bracket outside of the current.

Reply:
REF stands for reference instance . it is our internal "support server". so not much interest in which version of wicked is running there